@css/main_style.css

```diff
@@ -303,11 +796,122 @@ div.main-body {
 }
 
 #section_2 {
-  background-image: url("../@images/section-2_background.jpeg");
   background-attachment: scroll;
   background-repeat: no-repeat;
   background-size: cover;
   height: 588px;
   padding-left: 40px;
   padding-right: 40px;
 }
+
+#about-myself-layout {
+  padding: 0 0;
+  gap: 24px;
+}
+
+#about-myself {
+  padding: 0 24px 114px 24px;
+  align-items: center;
+  gap: 88px;
+}
+
+#contact-me {
+  background-image: url("../@images/contact-me_background.svg");
+  background-attachment: scroll;
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 375px;
+  height: 779px;
+  padding-left: 40px;
+  padding-right: 40px;
+  flex-shrink: 0;
+}
+
+#work-cards-section {
+  grid-template-rows: repeat(4, 1fr);
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: var(--separator-height);
+  padding: 114px 24px;
+  flex-direction: column;
+  align-items: center;
+  gap: 88px;
+}
+
+#work_card_1 {
+  padding-left: var(--workcard-padding-left);
+  padding-right: var(--workcard-padding-right);
+}
+
+#work_card_2 {
+  padding-left: var(--workcard-padding-left);
+  padding-right: var(--workcard-padding-right);
+}
+
+#work_card_3 {
+  padding-left: var(--workcard-padding-left);
+  padding-right: var(--workcard-padding-right);
+}
+
+#work_card_4 {
+  padding-left: var(--workcard-padding-left);
+  padding-right: var(--workcard-padding-right);
+}
+
+#social_logo_1a {
+  width: 18px;
+  height: 20px;
+  flex-shrink: 0;
+}
+
+#social_logo_2a {
+  width: 20px;
+  height: 20px;
+  flex-shrink: 0;
+}
+
+#social_logo_3a {
+  width: 24px;
+  height: 24px;
+}
+
+#social_logo_4a {
+  width: 20px;
+  height: 19.967px;
+  flex-shrink: 0;
+}
+
+#social_logo_5a {
+  width: 20px;
+  height: 20px;
+  flex-shrink: 0;
+}
+
+#languages {
+  width: 237px;
+  color: #000;			/* #000000 */
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;	/* 120% */
+}
+
+#frameworks {
+  width: 237px;
+  color: #000;			/* #000000 */
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;	/* 120% */
+}
+
+#skills {
+  width: 237px;
```

Comment:
- [ ] Congratulations on your excellent work! 👍 I noticed that using a fixed width has resulted in your design being unresponsive. 
- [ ] To address this, I would suggest considering the use of percentage-based widths instead. By doing so, you can ensure that your design becomes more responsive and adaptable to different screen sizes. 
- [ ] Additionally, please ensure that your design aligns with the provided Figma design.
- [ ]  your design
![your](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/118761303/aef4a7c1-3797-4d06-9d27-7a8919899073)

- [ ] Figma 
![figma](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/118761303/a77ee19f-422a-4f68-8cdb-17839c76ade5)